.gitignore

```diff
@@ -4,6 +4,7 @@ psqldump
 .DS_Store
 Csv2rdf/out_rdf.ttl
 .history/*
-cantusdb/data/*
+cantusdb/data/raw/*.csv
+Cantusdb/data/deprecated/*
```

Comment:
It looks like a file in this directory made it into the PR? `cantusdb_01102024.csv`

Reply:
I will remove.